packages/insomnia-app/app/ui/components/rendered-text.tsx

```diff
@@ -1,15 +1,18 @@
-import React, { PureComponent } from 'react';
+import React, { FC, PureComponent } from 'react';
+
+import { HandleRender } from '../../common/render';
+import { useNunjucks } from '../context/nunjucks';
 
 interface Props {
   children: string;
-  render: (...args: any[]) => any;
+  render: HandleRender;
 }
 interface State {
   renderedText: string;
   error: string;
 }
 
-export class RenderedText extends PureComponent<Props, State> {
+class InternalRenderedText extends PureComponent<Props, State> {
```

Comment:
```suggestion
class RenderedTextInternal extends PureComponent<Props, State> {
```

there are 6+ others like this that all put `Internal` at the end.  If we go with this approach for this PR I think we should keep the name consistent here with those others.

Reply:
47318dd30362a626ca684f3584da56bd552e75b6